### ISSUE TITLE
Add types for primaryKeyAttributes into Model class

### ISSUE THREAD

**Issue by nestorvanz** [sequelize/sequelize#11392]
Please add `primaryKeyAttributes` to Model class to allow access to primaryKeyAttributes from TypeScript without errors.

File: sequelize/types/lib/model.d.ts
```typescript
export abstract class Model<T = any, T2 = any> extends Hooks {
  /** The name of the database table */
  public static readonly tableName: string;

  /**
   * The name of the primary key attribute
   */
  public static readonly primaryKeyAttribute: string;

  /**
   * The name of the primary key attributes
   */
  public static readonly primaryKeyAttributes: string[];
```

**Comment by nainkunal933** [comment]
Hi, Can I work on this issue?

**Comment by papb** [comment]
@nainkunal933 Sure! Let me know if you need help.

**Comment by nainkunal933** [comment]
@papb I am working on the issue and I have two questions:

1) I have started the database containers using `docker-compose up postgres-95 mysql-57 mssql` but the tests are failing for mariadb. Contributing.md does not suggest any username or password for the database. 

![Snip20190908_8](https://user-images.githubusercontent.com/31940639/64496102-5f55f500-d256-11e9-9ab2-d724b837f1a9.png)

2) I have added support for multiple key attributes using a string[] as shown in the screenshot below. Do you expect any other format for the names of the primaryKeyAttributes?

![Snip20190908_7](https://user-images.githubusercontent.com/31940639/64496056-ed7dab80-d255-11e9-8909-a7ddef4a57a6.png)


If everything is done right then I will make a commit to the branch with the following commit message and open a PR. Thanks for your attention.  
 ```
feat: Added support for multiple primary key attributes

Added support for multiple primary key attributes in sequelize/types/lib/model.d.ts

Closes #11392
```

**Comment by papb** [comment]
@nainkunal933

For question 1 - I will investigate later. In the mean time, try to configure Travis CI in your fork to run the tests, should work.

For question 2 - I don't expect anything else. Feel free to open the PR!
